_posts/2023-10-26-install-vivaldi.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Installing Vivaldi
+description: Learn how to Install and set up Vivaldi in Vanilla OS. 
+date: 2023-10-26
+layout: article
+authors: 
+    - Phosphorus-M
+    - amsyarzero
+published: true
+---
+
+## Introduction
+
+[**Vivaldi**](https://vivaldi.com) is a freeware, cross-platform web browser based on Chromium, the open-source project behind Google Chrome. Vivaldi is intended to be a customizable, functional, and practical browser for the web.
+
+## Installing Vivaldi
+
+You can install Vivaldi using the following steps:
+
+First, download the latest version of Vivaldi from [**here**](https://vivaldi.com/download/).
+
+Execute the following command to access to the directory where the downloaded package is located:
+
+```bash
+cd ~/Downloads
+```
+
+Then, install the downloaded package into your subsystem using the following command:
+
+```bash
+apx install --sideload ./vivaldi-stable_*_amd64.deb
+```
+
+After the installation is complete, you must install the proprietary codecs needed for video and audio to play:
+
+```bash
+apx enter
+sudo apt update && sudo apt install chromium-codecs-ffmpeg-extra
+```
+
+## Installing Pipewire for Vivaldi
```

Comment:
This whole section is still part of the process of installing Vivaldi, so it should either be a subsection, or this header shouldn't be here at all.